neptune/alpha/internal/backends/hosted_neptune_backend.py

```diff
@@ -195,6 +201,18 @@ def create_experiment(self,
         except HTTPNotFound:
             raise ProjectNotFound(project_id=project_uuid)
 
+    @with_api_exceptions_handler
+    def create_checkpoint(self, notebook_id: uuid.UUID, jupyter_path: str) -> Optional[uuid.UUID]:
+        checkpoint = self.leaderboard_client.createEmptyCheckpoint(
+            notebookId=notebook_id,
+            checkpoint={
+                "path": jupyter_path
+            }
+        ).response().result
+        if checkpoint is not None:
```

Comment:
Why do we have to check it for None?

Reply:
for safety I suppose, It was like this in old implementation

Reply:
Api response was not checked in old implementation. If I am not mistaked, a return value from helper method `create_checkpoint` in `notebooks` module is what was checked for None. Still don't know why. But this None check have to be moved at least.